components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -846,6 +853,7 @@
 <config_AM_mocStreamfunction_enable ocn_grid="oRRS15to5">.false.</config_AM_mocStreamfunction_enable>
 <config_AM_mocStreamfunction_enable ocn_grid="oARRM60to10">.false.</config_AM_mocStreamfunction_enable>
 <config_AM_mocStreamfunction_enable ocn_grid="oARRM60to6">.false.</config_AM_mocStreamfunction_enable>
+<config_AM_mocStreamfunction_enable ocn_grid="EC30to60E2r2">.true.</config_AM_mocStreamfunction_enable>
```

Comment:
@jonbob are you sure you want this set to true? In the ocn log file I'm getting the following written at every hour:
```
0001-01-01_01:00:00  WC time:    1599851079.141
 ocn_moc_streamfunction AM: config_AM_mocStreamfunction_region_group not found in regionGroupNames
 ocn_moc_streamfunction AM: setting regionGroupNumber=1
 ocn_moc_streamfunction AM: config_AM_mocStreamfunction_transect_group not found in transectGroupNames
 ocn_moc_streamfunction AM: setting transectGroupNumber=1
```

Reply:
maybe not! I just saw that we have the file it would use, but that seems excessive. I'll go turn it off by default and anyone who wants that output can turn it on

Reply:
I agree this should be off by default.  It will mess with mpas-analysis which won't work well (will miss the GM bolus contribution.

Reply:
the MOC online AM does not add the GM bolus velocity contribution to the streamfunction calculation, so this should be off whenever GM is on.

Reply:
It should be off for the EC60to30v3 too btw. And instead it should be 'true' for the RRS30to10.

Reply:
Thanks @milenaveneziani for the clarification. We'll have to make this changes in another PR. Or one of us can make in issue for that so it doesn't get lost over the weekend -- just let me know if you have time to do it or else I will.

Reply:
@jonbob: just created an issue.

Reply:
Thanks @milenaveneziani . If you're happy with this PR, could you please add your review?